oss/Dockerfile

```diff
@@ -25,19 +25,21 @@ ENV JAVA_HOME /opt/java
 ENV JAVA_VERSION_MAJOR 8
 ENV JAVA_VERSION_MINOR 102
 ENV JAVA_VERSION_BUILD 14
-             
+ENV JAVA_CHECKSUM="50bc7ff61ba064c471adc2ec08e44690f0dac4cd673a3666b6d7b24a48bd7169"
```

Comment:
might be worth adding a note here (in both files) linking to https://www.oracle.com/webfolder/s/digest/8u102checksum.html to show where this came from, and that its actually a legit value